opentracing-spring-cloud-starter-jaeger/src/main/java/io/opentracing/contrib/spring/cloud/starter/jaeger/JaegerAutoConfiguration.java

```diff
@@ -55,23 +54,23 @@
 @EnableConfigurationProperties(JaegerConfigurationProperties.class)
 public class JaegerAutoConfiguration {
 
-  @Autowired(required = false)
-  private List<TracerBuilderCustomizer> tracerCustomizers = Collections.emptyList();
```

Comment:
Wouldn't moving this injection to the `tracer` method solve the unreliability issue, you're referring to, by making sure the beans are initialised in the correct order and as a result ObjectProvider wouldn't be required?